ReleaseTooling/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -170,15 +196,18 @@ enum CocoaPodUtils {
     case let .success(output):
       // Print the output to the console and return the information for all installed pods.
       print(output)
-      return installedPodsInfo(inProjectDir: directory)
+      // FIXME: Broken
```

Comment:
FIXME

Reply:
Thankfully outdated and the comment can be deleted, thanks! Will do.